spec/features/card_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe "the review process" do
+  before :each do
+    @card = create(:card)
+    @card.update_attributes(review_date: Date.today)
+    visit new_review_path
+  end
+
+  it "shows translated text" do
+    expect(page).to have_content @card.translated_text
+  end
+
+  it "shows 'Правильно!' when answer is correct" do
+    fill_in "review_user_input", with: " TeXt 1123 МиРуМ   "
+    click_on 'Review'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.